src/Chapter1.hs

```diff
@@ -653,7 +672,15 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit n =
+    if
+      mod n 10 == div n 1
```

Comment:
`div n 1` is the same as just `n`, so you can use that directly 🙂 

Also, you need to add additional thing in order for this function to work on negative input 🙂